cmake/thirdparty/SetupThirdParty.cmake

```diff
@@ -47,11 +47,14 @@
 ################################
 # Git
 ################################
-find_package(Git)
-if (Git_FOUND)
-  set(GIT_FOUND TRUE)
-  message(STATUS "Git Executable: " ${GIT_EXECUTABLE} )
-  message(STATUS "Git Version: " ${GIT_VERSION_STRING} )
+message(STATUS "Git Support is ${ENABLE_GIT}")
+if (ENABLE_GIT)
+    find_package(Git)
+    if (Git_FOUND)
+        set(GIT_FOUND TRUE)
+        message(STATUS "Git Executable: " ${GIT_EXECUTABLE} )
+        message(STATUS "Git Version: " ${GIT_VERSION_STRING} )
+    endif()
```

Comment:
Please add an ``else()`` clause that says we couldn't find git, since the user requested it (perhaps by default), and the message above says that "Git Support is ON"